mxfusion/modules/gp_modules/svgp_regression.py

```diff
@@ -198,6 +200,7 @@ def compute(self, F, variables):
         :returns: the outcome of the inference algorithm
         :rtype: mxnet.ndarray.ndarray.NDArray or mxnet.symbol.symbol.Symbol
         """
+        has_mean = self.model.F.factor.has_mean
```

Comment:
There is quite an overlap in terms of code between SVGPRegressionMeanVariancePrediction.compute and SVGPRegressionSamplingPrediction.compute. Maybe using some auxiliary functions to reuse some code would improve the readability.